app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -70,6 +70,7 @@ protected function validator(array $data)
             'first_name' => 'required|max:255',
             'email' => 'required|email|max:255|unique:users',
             'password' => 'required|min:6|confirmed',
+            'gdpr' => 'required',
```

Comment:
You should not call every variables like "gdpr". OK, the aim of ths PR is to solve the GDPR question, but this functionnnality is more than that. What if an other government create a new law ? You want to have an other variable with the second law's name ?